07LessonSeven/in-class-project/index.html

```diff
@@ -1,10 +1,72 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Template</title>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Nasa</title>
+
+    <link rel="stylesheet" href="css/my-style.css" type="text.css" />
```

Comment:
if you're wondering why your stylesheet wasn't being applied, you have type="text.css" here when it's text/css  :)